### PR TITLE
Add io.github.diegoivan.GaticTacToe

### PR DESCRIPTION
**Pull request by Diego-Ivan**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. 
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.GaticTacToe

File: io.github.diegoivan.GaticTacToe.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.vala"
+  ],
+  "build-options": {
+    "prepend-path": "/usr/lib/sdk/vala/bin/",
+    "prepend-ld-library-path": "/usr/lib/sdk/vala/lib"
+  },
```
Comment (hfiguiere) [inline io.github.diegoivan.GaticTacToe.json:12]: you don't need this, valac is in the Sdk.

The vala sdk extension is for IDE.

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.GaticTacToe

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.GaticTacToe

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.GaticTacToe

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.GaticTacToe

**Comment by Diego-Ivan** [comment]
I'm not very sure why the build is failing. I don't exactly what this error is about:
```
stat builddir/*/share/app-info/xmls/io.github.diegoivan.GaticTacToe.xml.gz
 in dir /srv/buildbot/worker/build-x86_64-6/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'stat builddir/*/share/app-info/xmls/io.github.diegoivan.GaticTacToe.xml.gz'
 using PTY: False
stat: cannot stat 'builddir/*/share/app-info/xmls/io.github.diegoivan.GaticTacToe.xml.gz': No such file or directory
program finished with exit code 1
elapsedTime=0.010660
```
Can I get a little help?

**Comment by Diego-Ivan** [comment]
@hfiguiere Any ideas on what might not be going right? Thanks in advance

**Comment by hfiguiere** [comment]
the appstream file is simply not found.

**Comment by hfiguiere** [comment]
```
WARNING: Failed to validate desktop file /var/home/hub/source/flathub/io.github.diegoivan.GaticTacToe/repo/export/share/applications/io.github.diegoivan.GaticTacToe.desktop: /var/home/hub/source/flathub/io.github.diegoivan.GaticTacToe/repo/export/share/applications/io.github.diegoivan.GaticTacToe.desktop: error: value "Games;GNOME;GTK;" for key "Categories" in group "Desktop Entry" contains an unregistered value "Games"; values extending the format should start with "X-"
/var/home/hub/source/flathub/io.github.diegoivan.GaticTacToe/repo/export/share/applications/io.github.diegoivan.GaticTacToe.desktop: hint: value "Games;GNOME;GTK;" for key "Categories" in group "Desktop Entry" does not contain a registered main category; application might only show up in a "catch-all" section of the application menu
```

This is building locally.

**Comment by hfiguiere** [comment]
Here is the list of main categories

https://specifications.freedesktop.org/menu-spec/latest/apa.html#main-category-registry

**Comment by Diego-Ivan** [comment]
Thank you so much for your help!

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.GaticTacToe

**Comment by hfiguiere** [comment]
it will fail.

the app-id in the appdata file is incorrect.

**Comment by hfiguiere** [comment]
build should have failed. But for sure the app-id in the appdata file doesn't match the app-id.

**Comment by Diego-Ivan** [comment]
Thanks for you comments! I'll push the changes and rebuild

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.GaticTacToe

**Pull request closed without merging**